### DISCUSSION TITLE
Move tags in their own field

### DISCUSSION THREAD

**Discussion by Lerzu** [bluesky-social/atproto#2842] [Bluesky Client App]
Currently when someone uses a lot of tags, it fills the text field which ends up looking like spam and often takes a large amount of space on the screen. Moving tags in their own, smaller field would allow users to use more tags to make their posts easier to find.

By default this would only show the first few tags (in the order you type them in), taking 1 line under the post, but clicking/tapping on it would open the full tag list that you can then select from which tag to further search from if needed. 20 or 30 hard limit on tags might be needed so the list doesn't go on forever.

Not everyone currently uses tags so to properly get something like this rolling, it would need some promoting on the platform if this becomes a feature. The way Feeds currently work aren't doing a good job of showing me what I want to see so that's another reason why tags need more attention to make manual searches more effective.

**Comment by MasterJ93** [comment]
There is, indeed, a separate field that's specifically for tags. It's limited at this time (I believe there's a maximum for 8 tags that you can add). Third-party clients have most likely exposed it, as well as API libraries. Mines does. The Bluesky app just hasn't exposed it publicly as of yet. Not too sure why.

**Reply by Lerzu** [reply]
Ohh, interesting. Hopefully that gets expanded so everyone can use it.
